k8s/src/main/scala/io/buoyant/k8s/PortMapLogger.scala

```diff
@@ -0,0 +1,33 @@
+package io.buoyant.k8s
+
+import com.twitter.logging.Logger
+
+private[k8s] case class PortMapLogger(nsName: String, serviceName: String) {
+  def logDiff[A,B](oldPorts: Map[A, B], newPorts: Map[A, B]): Unit =
+    if (log.isLoggable(Logger.TRACE)) {
+      for {
+        port <- oldPorts.keySet
+        oldValue <- oldPorts.get(port)
+      } newPorts.get(port) match {
+        case Some(`oldValue`) =>
```

Comment:
this is pretty confusing.  would prefer `case Some(value) if value == oldValue =>`

Reply:
yeah, was going to mention on your earlier comment that if this wasn't immediately clear, it probably ought to be changed.